enterprise/server/api/api_server_test.go

```diff
@@ -441,6 +457,53 @@ func getEnvAndCtx(t *testing.T, user string) (*testenv.TestEnv, context.Context)
 	return te, ctx
 }
 
+func streamBuildFromTestData(t *testing.T, te *testenv.TestEnv, testDataFile string) string {
```

Comment:
how do we generate this data format? Is this the same format produced by the "download json" button?